src/Compilers/CSharp/Portable/Lowering/Extensions.cs

```diff
@@ -96,15 +96,23 @@ public static bool NullableNeverHasValue(this BoundExpression expr)
                 return creation.Constructor.ParameterCount == 0;
             }
 
+            if (expr is BoundDefaultExpression)
```

Comment:
>if (expr is BoundDefaultExpression) [](start = 12, length = 35)

It looks like we already handled this situation in the ```if``` before the previous one.